modules/jpg/image_loader_jpegd.cpp

```diff
@@ -117,8 +117,6 @@ Error ImageLoaderJPG::load_image(Ref<Image> p_image, FileAccess *f, bool p_force
 
 	Error err = jpeg_load_image_from_buffer(p_image.ptr(), w.ptr(), src_image_len);
 
-	w = PoolVector<uint8_t>::Write();
```

Comment:
I think this actually does something behind the scene than just assigning a new value to `w`, so I don't know if it can be removed. CC @hpvb @karroffel

Reply:
It looks to me like all this does is reduce a reference count on the thing pointed to by `w`, which will happen anyway when w is destructed, even if the assignment isn't performed.

Reply:
I think it would be best to leave those as is in this PR, and let them be handled in #30206.